app/controllers/api/v1/uploads_controller.rb

```diff
@@ -1,17 +1,22 @@
 class Api::V1::UploadsController < ApplicationController
   def index
+    user = User.find(params[:userid])
```

Comment:
what if the user record does not exist and you get a RecordNotFound exception? where are you handling it?